src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -2819,7 +2819,7 @@ private void DoWriteError(object obj)
         /// but the command failure will ultimately be
         /// <see cref="System.Management.Automation.ActionPreferenceStopException"/>,
         /// </remarks>
-        internal void _WriteErrorSkipAllowCheck(ErrorRecord errorRecord, ActionPreference? actionPreference = null, bool isNativeError = false)
+        internal void _WriteErrorSkipAllowCheck(ErrorRecord errorRecord, ActionPreference? actionPreference = null, bool isNativeStdErrCall = false)
```

Comment:
isNativeStdErrCall is not informative enough to understand what it is for.
Do we need it till the feature is experimental? If so I'd want to see the fact be in the name.

Reply:
Looking through https://github.com/PowerShell/PowerShell/pull/13361 and https://github.com/PowerShell/PowerShell/pull/15864, I think the change here is the right one.

The name of the boolean parameter also reflects the specific circumstances of the call.

If you've got a specific suggestion here, please let me know